extensions/wikia/CommunityPage/scripts/CommunityPageBenefitsModal.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Modal containing Benefits of joining Community;
+ * modal is an entry point for Community Page
+ */
+define('CommunityPageBenefitsModal',
+	['wikia.loader', 'mw', 'wikia.mustache'],
+	function (loader, mw, mustache) {
+		'use strict';
+		var modalConfig = {
+				vars: {
+					id: 'CommunityPageBenefitsModal',
+					classes: ['community-page-benefits-modal'],
+					size: 'content-size'
+				}
+			},
+			specialCommunityTitle = new mw.Title('Community', -1),
+			wikiTopic = mw.config.get('wgSiteName');
```

Comment:
Why are `specialCommunityTitle` and `wikiTopic` global to the module when they are only used in `handleRequestsForModal`? Unless they need to be used in other methods, I'd just declare them in `handleRequestsForModal`.